<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" ignored="0" total="6" passed="4">
  <reporter-output>
    <line>
      <![CDATA[<br>Verification failure : did not expect to find [abc] but found [xyz]<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Sep_09_19_42_46_IST_2018.jpg><img src=Sun_Sep_09_19_42_46_IST_2018.jpg height=200 width=200></img></a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[Login sucessfully executed]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="4246" started-at="2018-09-09T19:42:46 IST" finished-at="2018-09-09T19:42:50 IST">
    <groups>
    </groups>
    <test name="Bank Manager Test" duration-ms="878" started-at="2018-09-09T19:42:46 IST" finished-at="2018-09-09T19:42:46 IST">
      <class name="com.ddf.testcases.BankMangerLogin">
        <test-method status="PASS" signature="bankMangerLogin()[pri:0, instance:com.ddf.testcases.BankMangerLogin@1ed6388a]" name="bankMangerLogin" duration-ms="557" started-at="2018-09-09T19:42:46 IST" finished-at="2018-09-09T19:42:46 IST">
          <reporter-output>
            <line>
              <![CDATA[<br>Verification failure : did not expect to find [abc] but found [xyz]<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Sep_09_19_42_46_IST_2018.jpg><img src=Sun_Sep_09_19_42_46_IST_2018.jpg height=200 width=200></img></a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[Login sucessfully executed]]>
            </line>
          </reporter-output>
        </test-method> <!-- bankMangerLogin -->
      </class> <!-- com.ddf.testcases.BankMangerLogin -->
    </test> <!-- Bank Manager Test -->
    <test name="Add Customer Test" duration-ms="3074" started-at="2018-09-09T19:42:46 IST" finished-at="2018-09-09T19:42:50 IST">
      <class name="com.ddf.testcases.AddCustomerTest">
        <test-method status="PASS" signature="AddCustomer(java.util.Hashtable)[pri:0, instance:com.ddf.testcases.AddCustomerTest@13e344d]" name="AddCustomer" duration-ms="1775" started-at="2018-09-09T19:42:47 IST" data-provider="dp" finished-at="2018-09-09T19:42:48 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, alerttext=Customer added successfully, postcode=6789.0, lastname=dash, firstname=ram}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddCustomer -->
        <test-method status="PASS" signature="AddCustomer(java.util.Hashtable)[pri:0, instance:com.ddf.testcases.AddCustomerTest@13e344d]" name="AddCustomer" duration-ms="473" started-at="2018-09-09T19:42:48 IST" data-provider="dp" finished-at="2018-09-09T19:42:49 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, alerttext=Customer added successfully, postcode=44554.0, lastname=prasad, firstname=sam}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddCustomer -->
        <test-method status="SKIP" signature="AddCustomer(java.util.Hashtable)[pri:0, instance:com.ddf.testcases.AddCustomerTest@13e344d]" name="AddCustomer" duration-ms="1" started-at="2018-09-09T19:42:49 IST" data-provider="dp" finished-at="2018-09-09T19:42:49 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, alerttext=Customer added successfully, postcode=44554.0, lastname=mishra, firstname=hari}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at com.ddf.testcases.AddCustomerTest.AddCustomer(AddCustomerTest.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:658)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:792)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1103)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1145)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
at org.testng.TestNG.runSuites(TestNG.java:997)
at org.testng.TestNG.run(TestNG.java:965)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddCustomer -->
        <test-method status="PASS" signature="AddCustomer(java.util.Hashtable)[pri:0, instance:com.ddf.testcases.AddCustomerTest@13e344d]" name="AddCustomer" duration-ms="437" started-at="2018-09-09T19:42:49 IST" data-provider="dp" finished-at="2018-09-09T19:42:49 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, alerttext=Customer added successfully, postcode=44554.0, lastname=patel, firstname=gopal}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddCustomer -->
      </class> <!-- com.ddf.testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
    <test name="Open Account Test" duration-ms="79" started-at="2018-09-09T19:42:50 IST" finished-at="2018-09-09T19:42:50 IST">
      <class name="com.ddf.testcases.OpenAccountTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.ddf.testcases.OpenAccountTest@1e6a3214]" name="setUp" is-config="true" duration-ms="12149" started-at="2018-09-09T19:42:33 IST" finished-at="2018-09-09T19:42:46 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="OpenAccount(java.util.Hashtable)[pri:0, instance:com.ddf.testcases.OpenAccountTest@1e6a3214]" name="OpenAccount" duration-ms="4" started-at="2018-09-09T19:42:50 IST" data-provider="dp" finished-at="2018-09-09T19:42:50 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Currency=Rupee, customer=ram dash}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the testSkipped the test case as run mode is set to NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the testSkipped the test case as run mode is set to NO
at com.ddf.testcases.OpenAccountTest.OpenAccount(OpenAccountTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:658)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:792)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1103)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:140)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:392)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:354)
at org.testng.SuiteRunner.run(SuiteRunner.java:302)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1145)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
at org.testng.TestNG.runSuites(TestNG.java:997)
at org.testng.TestNG.run(TestNG.java:965)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpenAccount -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ddf.testcases.OpenAccountTest@1e6a3214]" name="tearDown" is-config="true" duration-ms="1125" started-at="2018-09-09T19:42:50 IST" finished-at="2018-09-09T19:42:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ddf.testcases.OpenAccountTest -->
    </test> <!-- Open Account Test -->
  </suite> <!-- Suite -->
</testng-results>
